src/oci.h

```diff
@@ -184,7 +184,7 @@ struct oci_cfg_root {
 struct oci_cfg_user {
 	uid_t    uid; /*!< User ID to run workload as. */
 	gid_t    gid; /*!< Group ID to run workload as. */
-	gid_t  **additionalGids; /*!< extra Group IDs to set workload as a member of */
+	gchar  **additionalGids; /*!< extra Group IDs to set workload as a member of */
```

Comment:
It would be worth adding a comment explaining why additional groups are stored as strings whereas the primary group is a numeric.